pytest-pyramid-server/pytest_pyramid_server.py

```diff
@@ -107,6 +107,11 @@ def pre_setup(self):
         with open(str(self.working_config), 'w') as fp:
             parser.write(fp)
 
+        try:
+            parser.get('app:main', 'url_prefix')
+        except:
```

Comment:
Don't use bare `except`, it will catch things like OutOfMemory and so on - instead catch the specific exception that you care about